apis/v1alpha1/generator.yaml

```diff
@@ -1,2 +1,30 @@
-ignore:
-  resource_names:
+operations:
+  RequestCertificate:
+    resource_name: Certificate
+    operation_type: CREATE
+    override_values:
+      # NOTE(jaypipes): We only support DNS-based validation, because
+      # certificate renewal is not really automatable when email verification
+      # is used.
+      #
+      # See discussion here:
+      # https://docs.aws.amazon.com/acm/latest/userguide/email-validation.html
+      ValidationMethod: DNS
+  # NOTE(jaypipes): There is a GetCertificate API call, but that returns the
+  # actual cert bytes, not the attributes of the certificate request
+  DescribeCertificate:
+    resource_name: Certificate
+    operation_type: READ_ONE
+  UpdateCertificateOptions:
+    resource_name: Certificate
+    operation_type: UPDATE
```

Comment:
No delete operation? `DeleteCertificate`?

Reply:
DeleteCertificate is automatically inferred.